imod/mf6/simulation.py

```diff
@@ -1003,7 +1012,12 @@ def split(self, submodel_labels: GridDataArray) -> Modflow6Simulation:
         if not any(flow_models) and not any(transport_models):
             raise ValueError("a simulation without any models cannot be split.")
 
-        original_models = get_models(self)
+        original_models = {**flow_models,** transport_models}
```

Comment:
I do not get our linter settings anymore: How is the space between ``**`` and ``transport_models`` not raising any syntax issues here? Perhaps we should start working on setting ruff settings stricter and letting ruff format?

Reply:
You're right this is not supposed to happen. 
We now use the ruff linting for the linting task and ruff formatting for formatting. There seems to be a discrepancy between the linter and the formatter. 

We can replace the linting step with format --check . which should be aligned with the formatter.
I will create a separate issue for this

Reply:
removed that space manually

Reply:
I created issue #934 to fix this